python/arcticdb/util/hypothesis.py

```diff
@@ -64,6 +64,13 @@ def non_zero_and_non_infinite(x):
 def integral_type_strategies(draw):
     return draw(from_dtype(draw(st.one_of([unsigned_integer_dtypes(), integer_dtypes()]))).filter(non_infinite))
 
+@st.composite
+def signed_integral_type_strategies(draw):
```

Comment:
Do we need a utility method for this?

Isn't `integer_dtypes().filter(non_infinite)` sufficient?

Reply:
Removed the one_of, since it is totally unnecessary provided we only have one option now. The rest of logic seems to be necessary and I think it aligns to the rest of functions